delivery_state/data/mail_template.xml

```diff
@@ -5,49 +5,64 @@
         <field name="model_id" ref="delivery.model_stock_picking" />
         <field
             name="partner_to"
-        >${object.sale_id.partner_id.id or object.partner_id.id}</field>
+        >{{object.sale_id.partner_id.id or object.partner_id.id}}</field>
```

Comment:
I think this template is no longer needed since several versions, as Odoo includes a similar one, and the automatism.

Reply:
Yes, I can't see the difference right now: https://github.com/odoo/odoo/blob/15.0/addons/stock/data/mail_template_data.xml#L3